Task_1/FeTS_Challenge.ipynb

```diff
@@ -704,13 +704,68 @@
    "source": [
     "scores_dataframe"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Produce NIfTI files for best model outputs on the validation set\n",
+    "Now we will produce model outputs to submit to the leader board.\n",
+    "\n",
+    "At the end of every experiment, the best model (according to average ET, TC, WT DICE) \n",
+    "is saved to disk at: ~/.local/workspace/checkpoint/\\<checkpoint folder\\>/best_model.pkl,\n",
+    "where \\<checkpoint folder\\> is the one printed to stdout during the start of the \n",
+    "experiment (look for the log entry: \"Created experiment folder experiment_##...\" above)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from fets_challenge import model_outputs_to_disc\n",
+    "\n",
+    "# you will need to specify the correct experiment folder and the parent directory for\n",
+    "# the data you want to run inference over\n",
+    "checkpoint_folder='experiment_1'\n",
+    "data_path = '/raid/edwardsb/validation_set'\n",
```

Comment:
Maybe make this clear that this is the validation data downloaded from the challenge? i.e.

```
data_path = /PATH/TO/CHALLENGE_VALIDATION_DATA
```

Reply:
Done